docs/API/InstrumentFormat.md

```diff
@@ -287,32 +342,188 @@ A NumericElement represents a numeric data input and has the general form of:
 
 
 
-### 2.1.5: ScoreFieldElement
+### 2.1.5: CalcFieldElement
 
-A score field represents a placeholder to display/save values based on other
+A calc field represents a placeholder to display/save values based on other
 data entered by the user in the instrument but does not directly get input from
-the user. It has the following form.
+the user. It is displayed as a disabled text box. It has the following form.
 
 ```js
 {
-    "Type": "score",
+    "Type": "calc",
     "Name": REQUIRED,
     "Description": OPTIONAL,
+    "Formula" : string, /* LORIS Logic Parser formula */
     "Options": {
         /* None currently */
     }
 }
 ```
 
-`Type`: MUST be "score".
+`Type`: MUST be "calc".
 
-`Name`: Required. Follows `PageElement.Name` rules. The Name MAY be used by an
+`Name`: Required. Follows `PageElement.Name` rules. The Name will be used by an
         implementation as a field name to save calculated data into.
 
-`Description`: Optional. Follows `PageElement.Name` rules. If not specified, the
-               score will be displayed with no accompagning text.
+`Description`: Optional. If not specified, the
+               score will be displayed with no accompanying text.
+               
+`Formula`: String. Evaluated using the LORIS Logic Parser syntax.
+           The result is stored as the element's value.
+           
+### 2.1.6 Radio
+
+A radio element represents a set of radio buttons all attributed to the same question.
+A general example is provided below:
+
+```js
+{
+    "Type": "radio",
+    "Name": REQUIRED,
+    "Description": OPTIONAL,
+    "Options": {
+        "Values" : {
+            "v0" : "label0",
+            "v1" : "label1",
+            "v2" : "label2",
+            "v9" : "label9"
+        },
+        "Order" : {
+            "0" : "v9",
+            "1" : "v2",
+            "2" : "v1",
+            "3" : "v0"
+        },
+        "Orientation" : "horizontal|vertical",
+        "RequireResponse" : boolean
+    }
+}
+```
+
+`Type`: MUST be "radio".
+
+`Name`: Required. Follows `PageElement.Name` rules.
 
+`Description`: Optional. If not specified, the
+               radio buttons will be displayed with no accompanying text.
+               If specified, they will be displayed to the left of the buttons.
+               
+`Options.Values`: KEY REQUIRED, VALUE OPTIONAL. Contains a JSON object specifying options,
```

Comment:
I don't see how you can have a JSON key without a value. This sentence is confusing.

Reply:
What I meant was the value can be an empty string.